test/asynchronous/test_encryption.py

```diff
@@ -2499,6 +2502,7 @@ async def test_1_csfle_joins_no_schema(self):
             )
         )
         self.assertEqual(doc, {"csfle": "csfle", "matched": [{"no_schema": "no_schema"}]})
+        await encrypted_client.close()
```

Comment:
Closing the clients in the tests isn't needed since they're closed at tear down anyway. The reason I asked to close the clients in asyncSetUp is because otherwise they remain open for the entire test and compete with other tasks that need to be scheduled.

Reply:
ahh okay, i see. fixed